### PR TITLE
Added file size to import button

### PR DESCRIPTION
**Pull request by NuarkNoir**
It is done on CivitAI site, so why not here?
Also generaly it is much better to know how much data you have to download

### CODE REVIEW

File: StabilityMatrix/App.xaml.cs [hunk 1]
```diff
@@ -247,7 +247,9 @@ private void App_OnStartup(object sender, StartupEventArgs e)
                 });
 
             // Add Refit clients
-            serviceCollection.AddRefitClient<ICivitApi>(defaultRefitSettings)
+            // @NuarkNoir: When passing config to this method we are loosing sizeKB from response, who knows why.
+            // @NuarkNoit: Another solution, anyone?
+            serviceCollection.AddRefitClient<ICivitApi>()
```
Comment (mohnjiles) [inline StabilityMatrix/App.xaml.cs]: Hi, first of all, thanks for the PR!

The issue here is that we set the JsonPropertyName wrong in the `CivitFile.cs` class. If you change that file as such:
```diff
- [JsonPropertyName("sizeKb")]
+ [JsonPropertyName("sizeKB")]
public double SizeKb { get; set; }
```

it should work and you can add the `defaultRefitSettings` back to this here.

File: StabilityMatrix/Models/FileSizeType.cs [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+using System.Threading.Tasks;
+using System.Windows;
+using System.Windows.Media.Imaging;
+using System.Windows.Threading;
+using AsyncAwaitBestPractices;
+using CommunityToolkit.Mvvm.ComponentModel;
+using CommunityToolkit.Mvvm.Input;
+using NLog;
+using StabilityMatrix.Extensions;
+using StabilityMatrix.Helper;
```
Comment (mohnjiles) [inline StabilityMatrix/Models/FileSizeType.cs]: Could you remove the unused using statements here please?

File: StabilityMatrix/Models/FileSizeType.cs [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    {
+        SizeInKB = sizeInKB;
+        HumanReadableRepresentation = ConvertToHumanReadable();
+    }
+
+    private string ConvertToHumanReadable()
+    {
+        string[] sizeUnits = { "KB", "MB", "GB", "TB" };
+        double size = SizeInKB;
+        int unitIndex = 0;
```
Comment (mohnjiles) [inline StabilityMatrix/Models/FileSizeType.cs]: Could you use `var` where applicable here please?

**Review by mohnjiles — CHANGES_REQUESTED** [review]

**Comment by mohnjiles** [comment]
Thanks again for the PR! We'll get a code style guideline setup soon so you don't have to guess at those kinds of things in the future 😄

File: StabilityMatrix/Models/FileSizeType.cs (hunk 2 above)
Comment (NuarkNoir) [reply StabilityMatrix/Models/FileSizeType.cs]: Oh, sorry, my bad

File: StabilityMatrix/Models/FileSizeType.cs (hunk 3 above)
Comment (NuarkNoir) [reply StabilityMatrix/Models/FileSizeType.cs]: Okay

File: StabilityMatrix/App.xaml.cs (hunk 1 above)
Comment (NuarkNoir) [reply StabilityMatrix/App.xaml.cs]: 👌

**Comment by NuarkNoir** [comment]
Now all should be good

**Comment by ionite34** [comment]
Hi, thanks for the PR, all looks good. Could you update the branch with the latest main changes? I don't think there should be any conflicts.

**Comment by ionite34** [comment]
recheck

**Comment by NuarkNoir** [comment]
I have read the CLA Document and I hereby sign the CLA

**Review by ionite34 — APPROVED** [review]

**Review by mohnjiles — APPROVED** [review]

**Pull request merged**